helm/rucio-cron-jobs/templates/wfDashboard.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.wfDashboard.enabled -}}
```

Comment:
I think the content here is correct, but we've already merged a wf-dashboard cronjob into the code here under cron-jobs. So please remove that one and name this file wf-dashboard.yaml (lots of Kubernetes stuff follows the dash-separator naming scheme)